tests/openthread/Makefile

```diff
@@ -32,17 +47,12 @@ endif
 
 USEMODULE += $(DRIVER)
 
+USEMODULE += xtimer
```

Comment:
No need to add xtimer module as a dependency here if it's already a dependency of the openthread package.

Reply:
Still there.